instance/models/mixins/openedx_config.py

```diff
@@ -403,6 +402,13 @@ def _get_configuration_variables(self):
                 "PRIVACY": self.privacy_policy_url,
             }
 
+        # use the new forum heartbeat path if newer than ironwood.1
+        if 'ironwood' in self.openedx_release and 'ironwood.1' not in self.openedx_release:
```

Comment:
Continuing the discussion in https://github.com/open-craft/opencraft/pull/463#discussion_r322669445 here.

>  It's a bit messy because it appears the path changed between ironwood.1 and ironwood.2. Is this the kind of conditional checking you'd like to see?

@swalladge, I checked the files and the commit history in the upstream `open-release/ironwood.master` branch and also our `opencraft-release/ironwood.2` branch and could not find the change to move the `check_forum_heartbeat` function from `lms/lib/comment_client/utils.py` to `openedx/core/djangoapps/django_comment_common/comment_client/utils.py`. So it looks like the path change is needed only for the master branch and the PRs based on it as the change was not backported to Ironwood for obvious reasons. Did I miss something?

Reply:
Ah true. I had just assumed that because that change was in master before ironwood.2 was released, that it would be part of ironwood.2. That's not the case though. The change to only read from the yml files instead of json is only in master too.